api/team.js

```diff
@@ -157,6 +157,51 @@ router.post("/join/:code", async (req, res) => {
   });
 });
 
+// POST /api/team/join/:code
+// - If a hacker is not on a team, attempt to join a team
+router.post("/kick/:userid", async (req, res) => {
+  const hackerProfile = await models.HackerProfile.findOne({
+    where: { userId: req.user.id }
+  });
+
+  // Can't join a team if you're already on one!
```

Comment:
I am not sure I understand this comment, I think you meant that we can kick someone off of a team if they do not belong on any team themself?